torchvision/utils.py

```diff
@@ -88,8 +91,9 @@ def norm_range(t, range):
     return grid
 
 
-def save_image(tensor, fp, nrow=8, padding=2,
-               normalize=False, range=None, scale_each=False, pad_value=0, format=None):
+def save_image(tensor: Union[torch.Tensor, Sequence[torch.Tensor]], fp: Union[str, io.FileIO, io.BytesIO],
+               nrow: int = 8, padding: int = 2, normalize: bool = False, range: Optional[Tuple[int, int]] = None,
+               scale_each: bool = False, pad_value: int = 0, format: Optional[str, io.FileIO] = None) -> None:
```

Comment:
can format be a `FileIO`? From https://github.com/python-pillow/Pillow/blob/349bcd86915958f67b865f5c4c9ef683b7b46756/src/PIL/Image.py#L2094, looks like it calls `.upper()` on it.

Reply:
Nope, should be `Optional[str]`.